worthapp/index.html

```diff
@@ -1291,7 +1291,7 @@ <h1 class="display-3">
   Beginning the Journey
  </h3>
  <div class="embed-responsive embed-responsive-16by9">
-  <video class="embed-responsive-item" controls="" poster="img/lucia.png">
+  <video class="embed-responsive-item" controls="" poster="img/.jpg">
```

Comment:
Yeah, it's not obvious but what's going on here is `worthapp/index.html` is really just an output file that I'm committing, that's generated with the `make build` command, which pulls together all the templates in `scripts/templates/`